test/extended/gssapi.sh

```diff
@@ -0,0 +1,201 @@
+        --generator=run-pod/v1 --restart=Never --attach \
+        --env=SERVER="${server_config}" \
+        1> "${LOG_DIR}/${image_name}-${server_config}.log" \
+        2>> "${JUNIT_GSSAPI_OUTPUT}"
+    os::cmd::expect_success_and_text "cat '${LOG_DIR}/${image_name}-${server_config}.log'" 'SUCCESS'
+    os::cmd::expect_success_and_not_text "cat '${LOG_DIR}/${image_name}-${server_config}.log'" 'FAILURE'
+    os::cmd::try_until_success "oc get pod '${image_name}' -o jsonpath='${spec}'" # TODO fix this
+    os::cmd::expect_success_and_text "oc get pod '${image_name}' -o jsonpath='${spec}'" '0'
+    os::cmd::expect_success "oc delete pod ${image_name}"
+}
+
+trap "cleanup" EXIT
+
+echo "[INFO] Starting server"
+
+configure_os_server
+
+patch='
+{
+  "oauthConfig": {
+    "identityProviders": [{
+      "name": "header",
+      "challenge": true,
+      "mappingMethod": "add",
+      "provider": {
+        "apiVersion": "v1",
+        "kind": "RequestHeaderIdentityProvider",
+        "challengeURL": "http://gssapiproxy-server.gssapiproxy.svc.cluster.local/mod_auth/oauth/authorize?${query}",
+        "headers": ["Remote-User"]
+      }
+    }]
+  }
+}
+'
+cp "${SERVER_CONFIG_DIR}/master/master-config.yaml" "${SERVER_CONFIG_DIR}/master/master-config.tmp.yaml"
+openshift ex config patch "${SERVER_CONFIG_DIR}/master/master-config.tmp.yaml" --patch="${patch}" > "${SERVER_CONFIG_DIR}/master/master-config.yaml"
+USE_LATEST_IMAGES=true ## Is this needed?
+start_os_server
+
+KUBECONFIG="${ADMIN_KUBECONFIG}"
+
+install_registry
+wait_for_registry
+
+os::cmd::expect_success 'oc login -u system:admin'
+os::cmd::expect_success "oc new-project ${PROJECT}"
+os::cmd::expect_success "oadm policy add-scc-to-user anyuid -z default -n ${PROJECT}"
+
+# create all the resources we need
+cp -R test/extended/testdata/gssapi "${BASETMPDIR}"
+TEST_DATA="${BASETMPDIR}/gssapi"
+
+HOST='gssapiproxy-server.gssapiproxy.svc.cluster.local'
+REALM="${HOST^^}"
+BACKEND='https://openshift.default.svc.cluster.local:443'
+
+os::cmd::expect_success "oc create -f '${TEST_DATA}/proxy'"
+
+# kick off a build and wait for it to finish
+os::cmd::expect_success "oc set env dc/gssapiproxy-server HOST='${HOST}' REALM='${REALM}' BACKEND='${BACKEND}'"
+os::cmd::expect_success "oc start-build --from-dir='${TEST_DATA}/proxy' --follow gssapiproxy"
+
+OS_IMAGES=(fedora ubuntu)
+
+for os_image in "${OS_IMAGES[@]}"; do
+
+    cp "$(which oc)" "${TEST_DATA}/${os_image}/base"
+    cp -R "${OS_ROOT}/hack" "${TEST_DATA}/${os_image}/base"
+    cp "${TEST_DATA}/scripts/test-wrapper.sh" "${TEST_DATA}/${os_image}/base"
+    cp "${TEST_DATA}/scripts/gssapi-tests.sh" "${TEST_DATA}/${os_image}/base"
+    cp "${TEST_DATA}/config/config" "${TEST_DATA}/${os_image}/base"
+
+    pushd "${TEST_DATA}/${os_image}" > /dev/null
+        pushd base > /dev/null
+            docker build --build-arg REALM="${REALM}" --build-arg HOST="${HOST}" -t "${PROJECT}/${os_image}-gssapi-base:latest" .
```

Comment:
i'm going to speculate "yes" but to be honest i'm not quite certain, i'd have to check the binary build codepath to be certain.